jenkins/bootstrap_test.py

```diff
@@ -1071,20 +1071,24 @@ def testAllProjectAreUnique(self):
             'ci-kubernetes-kubemark-100-gce.sh': 'ci-kubernetes-kubemark-*',
             'ci-kubernetes-kubemark-5-gce.sh': 'ci-kubernetes-kubemark-*',
             'ci-kubernetes-kubemark-high-density-100-gce.sh': 'ci-kubernetes-kubemark-*',
+            'ci-kubernetes-e2e-gke-large-cluster.sh': 'ci-kubernetes-e2e-gke-large-*',
+            'ci-kubernetes-e2e-gke-large-deploy.sh': 'ci-kubernetes-e2e-gke-large-*',
+            'ci-kubernetes-e2e-gke-large-teardown.sh': 'ci-kubernetes-e2e-gke-large-*',
         }
         projects = collections.defaultdict(set)
         for job, job_path in self.jobs:
             with open(job_path) as fp:
                 lines = list(fp)
             for line in lines:
+                if line.startswith('#'):
```

Comment:
What is the motivation for this change?

I am pretty sure this has a bunch of unintended side effects for node jobs which do not put their variables into the job script.

https://github.com/kubernetes/test-infra/blob/master/jobs/ci-kubernetes-node-kubelet-1.4.sh#L21